C/CPUInfo/build_tarballs.jl

```diff
@@ -37,6 +41,7 @@ fi
 # platforms are passed in on the command line
 platforms = supported_platforms()
 filter!(p -> !(Sys.isapple(p) && arch(p) == "aarch64"), platforms) # aarch64-macos unsupported
+filter!(p -> !(Sys.iswindows(p) && arch(p) == "i686"), platforms) # Failing to link
```

Comment:
What's the problem?

Reply:
```
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x23): undefined reference to `InitOnceExecuteOnce'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x9b): undefined reference to `GetMaximumProcessorGroupCount'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0xdc): undefined reference to `GetMaximumProcessorCount'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x18b): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x271): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x2de): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] CMakeFiles/cpuinfo.dir/objects.a(init.c.obj):init.c:(.text+0x48d): undefined reference to `GetLogicalProcessorInformationEx'
[19:27:56] collect2: error: ld returned 1 exit status
[19:27:56] make[2]: *** [CMakeFiles/cpuinfo.dir/build.make:322: libcpuinfo.dll] Error 1
```
According to https://docs.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-initonceexecuteonce, `InitOnceExecuteOnce` needs linking to `kernel32.dll`

Reply:
Its fixed in the earlier released subsequent version 20200612, #4560, and again in #4617.

Reply:
… so maybe not worth the trouble?

Reply:
Ok, then I'll go ahead and merge this :+1: